src/_pytest/config/__init__.py

```diff
@@ -1650,6 +1651,66 @@ def getvalueorskip(self, name: str, path=None):
         """Deprecated, use getoption(skip=True) instead."""
         return self.getoption(name, skip=True)
 
+    #: Verbosity for failed assertions (see :confval:`verbosity_assertions`).
+    VERBOSITY_ASSERTIONS: Final = "assertions"
+    _KNOWN_VERBOSITY_TYPES: Final = {VERBOSITY_ASSERTIONS}
+    _VERBOSITY_INI_DEFAULT = "auto"
+
+    def get_verbosity(self, verbosity_type: Optional[str] = None) -> int:
+        r"""Access to fine-grained verbosity levels.
+
+        .. code-block:: ini
+
+            # content of pytest.ini
+            [pytest]
+            verbosity_assertions = 2
+
+        .. code-block:: console
+
+            pytest -v
+
+        .. code-block:: python
+
+            print(config.get_verbosity())  # 1
+            print(config.get_verbosity(Config.VERBOSITY_ASSERTIONS))  # 2
+        """
+        global_level = self.option.verbose
+        assert isinstance(global_level, int)
+        if (
+            verbosity_type is None
+            or verbosity_type not in Config._KNOWN_VERBOSITY_TYPES
+        ):
+            return global_level
+
+        level = self.getini(Config._ini_name(verbosity_type))
+
+        if level == Config._VERBOSITY_INI_DEFAULT:
+            return global_level
+
+        return int(level)
+
+    @staticmethod
```

Comment:
The name `_ini_name` is now too general, we should change it to `_verbosity_ini_name` or (my preference) inline it.